source/release-notes/1.4.txt

```diff
@@ -27,7 +25,54 @@ Starting in 1.4.0, you can set the workload level that
 destination clusters. For more information, see the
 :option:`--loadLevel` option.
 
+Improved Resiliency
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` 1.4.0 improves :term:`oplog` handling. The ``oplog``
+handling improvement adds resiliency to migrations and reduces the time
+to complete large migrations.
+
+Faster Processing for Sharded Clusters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Version 1.4.0 improves performance for replica sets and single-shard
+clusters. Multi-shard clusters have even better speed improvements.
+
+Telemetry
+~~~~~~~~~
+
+Starting in 1.4.0, ``mongosync`` collects anonymous, aggregated usage
+data to improve MongoDB products. You can disable collection of this
+:ref:`telemetry <c2c-telemetry>` data when you start ``mongosync``.
+
+Bug Fixes
+----------
+
+- ``REP-2148`` Fixes an issue so documents with ``_id`` values that
```

Comment:
I think we should get rid of the JIRA ticket IDs here. None of the other mongosync release notes have them, and the REP project isn't publicly accessible, so people can't actually use them to get more information.

Reply:
@mmcclimon I added them in for two reasons 

- One is because we have the issue numbers on the server release notes so this way matches the server approach. Those are public though so maybe they are more helpful there. 

- The other reason is more public facing. I thought perhaps it would be useful for customers/support to be able to refer to particular issues. 


No problem dropping them if you think that's better.